--- OpenCC-ver.1.0.6/CMakeLists.txt.orig	2020-04-12 17:22:50.000000000 +0200
+++ OpenCC-ver.1.0.6/CMakeLists.txt	2020-04-26 16:57:35.484655518 +0200
@@ -65,10 +65,10 @@
 ######## Directory
 
 set (DIR_PREFIX ${CMAKE_INSTALL_PREFIX})
-set (DIR_INCLUDE ${DIR_PREFIX}/include/)
-set (DIR_SHARE ${DIR_PREFIX}/share/)
-set (DIR_ETC ${DIR_PREFIX}/etc/)
-set (DIR_LIBRARY ${DIR_PREFIX}/lib${LIB_SUFFIX}/)
+set (DIR_INCLUDE ${DIR_PREFIX}/include)
+set (DIR_SHARE ${DIR_PREFIX}/share)
+set (DIR_ETC ${DIR_PREFIX}/etc)
+set (DIR_LIBRARY ${DIR_PREFIX}/lib${LIB_SUFFIX})
 
 if (DEFINED SHARE_INSTALL_PREFIX)
   set (DIR_SHARE ${SHARE_INSTALL_PREFIX})
@@ -86,8 +86,8 @@
   set (DIR_LIBRARY ${LIB_INSTALL_DIR})
 endif (DEFINED LIB_INSTALL_DIR)
 
-set (DIR_SHARE_OPENCC ${DIR_SHARE}/opencc/)
-set (DIR_SHARE_LOCALE ${DIR_SHARE}/locale/)
+set (DIR_SHARE_OPENCC ${DIR_SHARE}/opencc)
+set (DIR_SHARE_LOCALE ${DIR_SHARE}/locale)
 
 ######## Configuration
 
